### ISSUE TITLE
THEA withdrawal transactions not visible?

### ISSUE THREAD

**Issue by jaymansfield** [Polkadex-Substrate/Polkadex-Open-Beta#505]
THEA withdrawal transactions not visible?

I've submitted two withdrawals and neither are visible and its been 5+ minutes.

![image](https://github.com/Polkadex-Substrate/Polkadex-Open-Beta/assets/6218678/2d01bc9f-9b79-4009-a6e8-d349f6d5d40e)

**Comment by jaymansfield** [comment]
Now 15+ minutes and still can't see my transactions

**Comment by nuel77** [comment]
i tried the same and it worked. Did you try refreshing the transactions page? (make sure the same wallet you used for polkadex is connected)
<img width="1440" alt="image" src="https://github.com/Polkadex-Substrate/Polkadex-Open-Beta/assets/50878033/1dde4c3f-0083-4360-b1e8-6eae47d60d7a">

**Comment by jaymansfield** [comment]
@nuel77  yes many times. They just showed up now after over an hour later.

**Comment by pavankanteti** [comment]
Have you selected the wallet?

**Comment by HenryEnrique** [comment]
> @nuel77 yes many times. They just showed up now after over an hour later.

Same thing happened to me but thanks to Blackhathacker111 who helped me get recovered my lost funds,message him on telegram his telegram username is https://t.me/Blackhathacker110
